vsts/pipelines/ci.yml

```diff
@@ -76,32 +76,6 @@ stages:
         displayName: 'Set release tag name as environment variable'
       - template: templates/_signBinary.yml
 
-    #- job: Job_SignBinariesBuildServer
```

Comment:
Could you please clarify why you removed this from the pipeline? I would have expected you would have needed it.

Reply:
Yes, this code is basically referencing _signBinaryBuildServer.yml: https://github.com/microsoft/Oryx/pull/1268/files#diff-58841cda045384dd117e38a9271168bc0332c2776a5ca7330531275083371decL103

Which has the essential code from _signBinaryBuildServer.yml in the _signBinary.yml.

Reply:
- Signing binaries
- Archiving binaries
- Copying binaries
Are all in the same pipeline job with  _signBinary.yml

Reply:
OK thanks